Falanx.JsonCodec/paket.references

```diff
@@ -1 +1,2 @@
-Fleece.NewtonsoftJson
\ No newline at end of file
+Fleece.NewtonsoftJson
+Newtonsoft.Json
```

Comment:
Hmm so transient deps were not included

Reply:
no is ok, i just want to pin the version of newtonsoft, because is annoying